File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        n= set(nums)
        longest =0
        for i in nums:
            length =0
            if (i-1) not in n: 
                while (i+length) in n:
                    length+=1
            longest = max(longest, length)

        return longest

File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int s = nums.length;
        int b=0;
        int [] a= new int[s*2];
        for (int i=0;i<s*2;i++){
            if (b==s){
                b=0;
            }
            a[i]=nums[b];
            b++;
        }
        return a;
    }
}